src/main/kotlin/ink/abb/pogo/scraper/tasks/ProcessPokestops.kt

```diff
@@ -42,13 +44,22 @@ class ProcessPokestops(var pokestops: MutableCollection<Pokestop>) : Task {
             val distanceB = self.getEarthDistance(locationB)
             distanceA.compareTo(distanceB)
         })
-        if (startPokeStop == null)
-            startPokeStop = sortedPokestops.first()
+        if (startPokestop == null)
+            startPokestop = sortedPokestops.first()
 
         if (settings.lootPokestop) {
             val loot = LootOneNearbyPokestop(sortedPokestops, lootTimeouts)
             bot.task(loot)
         }
+        if (settings.campLurePokestop > 0) {
+            // TODO: grab that 70 from DOWNLOAD_SETTINGS
+            val luresInRange = sortedPokestops.filter { it.inRange(ctx.api, 70.0) && it.hasLure() }.size
+            if (luresInRange >= settings.campLurePokestop) {
+                Log.green("$luresInRange lures in range, pausing")
+                ctx.api.map.getMapObjects(3)
```

Comment:
@grover-c13 am I correct to assume that the API won't return new `CatchablePokemon` without this call (aka the cache is never invalidated by a timer)?